scholarx/archive/functions.js

```diff
@@ -52,13 +58,121 @@ async function getData() {
 }
 
 async function loadData() {
-    const payload = await getData();
-    const mentorProfiles = payload.mentor2021.concat(payload.mentor2020).concat(payload.mentor2019);
-    let mentor_Profiles = Mustache.render($("#templateMentors").html(), { "mentorProfiles": mentorProfiles });
-    $("#mentorProfiles").html(mentor_Profiles);
+    const { data } = await getData();
+    mentors = data.mentors;
+    mentees = data.mentees;
+    mentorProfilesData = mentors[2019].concat(mentors[2020]).concat(mentors[2021]);
+    let mentorProfiles = Mustache.render($("#templateMentors").html(), { "mentorProfiles": mentorProfilesData });
+    $("#mentorProfiles").html(mentorProfiles);
 
-    const menteeProfiles = payload.mentee2021.concat(payload.mentee2020).concat(payload.mentee2019);
-    let mentee_Profiles = Mustache.render($("#templateMentees").html(), { "menteeProfiles": menteeProfiles });
-    $("#menteeProfiles").html(mentee_Profiles);
+    menteeProfilesData = mentees[2019].concat(mentees[2020]).concat(mentees[2021]);
+    let menteeProfiles = Mustache.render($("#templateMentees").html(), { "menteeProfiles": menteeProfilesData });
+    $("#menteeProfiles").html(menteeProfiles);
 }
 loadData();
+function renderProfiles(mentorYear,menteeYear) {
+    let mentorProfiles = Mustache.render($("#templateMentors").html(), { "mentorProfiles": mentorYear });
+    let menteeProfiles = Mustache.render($("#templateMentees").html(), { "menteeProfiles": menteeYear });
+    $("#mentorProfiles").html(mentorProfiles);
+    $("#menteeProfiles").html(menteeProfiles);
+}
+function renderAllProfiles() {
+    let mentorProfiles = Mustache.render($("#templateMentors").html(), { "mentorProfiles": mentorProfilesData });
+    let menteeProfiles = Mustache.render($("#templateMentees").html(), { "menteeProfiles": menteeProfilesData });
+    $("#mentorProfiles").html(mentorProfiles);
+    $("#menteeProfiles").html(menteeProfiles);
+}
+//function to return true if all industry checkboxes are unchecked.
+function allFalse(){
```

Comment:
Let's use something like this:
```suggestion
function isEveryIndustryUnchecked(){
```